packages/stacking/src/utils.ts

```diff
@@ -140,7 +139,7 @@ export function poxAddressToBtcAddress(...args: PoxAddressArgs): string {
 }
 
 export function getBTCAddress(version: Buffer, checksum: Buffer) {
-  const btcAddress = address.toBase58Check(checksum, new BN(version).toNumber());
+  const btcAddress = address.toBase58Check(checksum, Number(version.toString()));
```

Comment:
This is a bit confusing. The `version` variable should always be a single byte right? In that case, wouldn't something like `address.toBase58Check(checksum, version[0])` make more sense?

Reply:
Great catch!
Added a [fix](https://github.com/hirosystems/stacks.js/pull/1150/commits/a1b65de96e104cdb328039e68907a8f972449b94) that also accepts plain number as `version`